os/include/tinyara/lcd/st7785.h

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ *
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __DRIVERS_LCD_ST7785_H
+#define __DRIVERS_LCD_ST7785_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <sys/types.h>
+#include <stdint.h>
+#include <tinyara/lcd/mipi_lcd.h>
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+/* System Function Command Table 1 */
+
+#define MIPI_DSI_RTNI           4
+#define MIPI_DSI_HSA            30
+#define MIPI_DSI_HBP            30
+#define MIPI_DSI_HFP            60
+#define MIPI_DSI_VSA            4
+#define MIPI_DSI_VBP            4
+#define MIPI_DSI_VFP            8
+#define MIPI_LCD_LIMIT		250
+#define MIPI_FRAME_RATE         60
+#define MIPI_LANE_NUMBER        1
+
+#if defined(CONFIG_LCD_PORTRAIT)
+#define LCD_XRES 240
+#define LCD_YRES 320
```

Comment:
There are CONFIG_LCD_XRES and CONFIG_LCD_YRES. Why do you define them for every orientation? Can't we conver them based on config_lcd_xres/yres and config_lcd_<orientation>?